mysql/assets/configuration/spec.yaml

```diff
@@ -255,6 +255,34 @@ files:
                   type: boolean
                   example: false
 
+              - name: table_size_metrics
+                description: |
+                  Set to `true` to collect data table size metrics.
+
+                  Note that only tables for which the user has been granted access privileges are collect.
```

Comment:
can you add some more color to this? e.g.:

it looks like this will only work if I add `GRANT SELECT ON <schema_name>.* TO 'datadog';`. I would just say this requires x specific permission to be granted to the datadog user in order for this metric collection to work